src/Livewire/Order/Order.php

```diff
@@ -407,6 +407,8 @@ public function save(): bool
         $this->order->address_delivery = $this->order->address_delivery ?: [];
         try {
             $action = UpdateOrder::make($this->order->toArray())->checkPermission()->validate();
+
+            $this->getAvailableStates('state');
```

Comment:
shouldn't the available states for delivery_state (and maybe payment_state) also be updated?